pkg/apis/sources/config/ping_defaults.go

```diff
@@ -17,21 +17,18 @@ limitations under the License.
 package config
 
 import (
-	"encoding/json"
 	"fmt"
+	"strconv"
 
 	corev1 "k8s.io/api/core/v1"
-	"sigs.k8s.io/yaml"
 )
 
 const (
 	// PingDefaultsConfigName is the name of config map for the default
 	// configs that pings should use.
-	PingDefaultsConfigName = "config-ping-webhook"
+	PingDefaultsConfigName = "config-ping-defaults"
 
-	// PingDefaulterKey is the key in the ConfigMap to get the name of the default
-	// Ping CRD.
-	PingDefaulterKey = "ping-config"
+	DataMaxSizeKey = "dataMaxSize"
 
 	PingDataMaxSize = -1
```

Comment:
While we're at it, shall we call that "**Default** DataMaxSize"?

Reply:
the configmap key? the internal variable name? Both?

Reply:
Just the variable name referring to the default value `-1`.

Reply:
(fwliw, if this gets merged in the meantime, I'll follow up with another commit addressing this feedback)